src/metabase/util/fonts.clj

```diff
@@ -0,0 +1,41 @@
+(ns metabase.util.fonts
+  "font loading functionality."
+  (:require [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [metabase.util.files :as u.files]))
+
+(def font-path
+  "font resources path as a String."
+  "./resources/frontend_client/app/fonts/")
+
+(defn- normalize-font-dirname
+  "Use a font's directory to derive a Display Name.
+
+  Underscores become spaces, and then split on dashes and take the first word.
+  This is done because of the Lato-v16-latin directory name."
```

Comment:
Curious, is there a reason the directory name couldn't just be changed to `Lato` to simplify the normalization required?

Reply:
Hm, good question. Maybe I had a bit of tunnel vision haha. I'll ask around.

Reply:
I think that would be a decent call given that the other fonts aren't that complicated. I'll need to change some URLs on the frontend, but that's easily done